.github/release-drafter.yml

```diff
@@ -0,0 +1,55 @@
+
+name-template: 'Liquibase v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+exclude-labels:
+  - 'skipReleaseNotes'
+categories:
+  - title: ':green_book: Notable Changes'
+    labels: 
+      - 'notableChanges'
+  - title: '🚀 New Features'
+    labels:
+      - 'TypeEnhancement'
+      - 'TypeTest'
+  - title: '🐛 Bug Fixes 🛠'
+    labels:
+      - 'TypeBug'
+  - title: '💥 Breaking Changes'
+    labels: 
+      - 'breakingChanges'
+  - title: '💥 API Breaking Changes'
+    labels: 
+      - 'APIBreakingChanges'
+  - title: '🤖 Security Driver and Other Updates'
+    collapse-after: 5
+    labels:
+      - 'sdou'
+      - 'dependencies'
+  - title: '👏 New Contributors'
+    labels:
+      - 'newContributors'
+       
+  
+change-template: '- (#$NUMBER) $TITLE @$AUTHOR '
+change-title-escapes: '\<*_&' # You can add # and @ to disable mentions, and add ` to disable code blocks.
+version-resolver:
+  major:
+    labels:
+      - 'major'
+  minor:
+    labels:
+      - 'minor'
+  patch:
+    labels:
+      - 'feature'
```

Comment:
for consistency purposes could we keep them as TypeBug, TypeEnhancement,etc. Just a suggestion. Feel free to ignore it

Reply:
I copied from https://github.com/liquibase/liquibase/blob/master/.github/release-drafter-master.yml, are they wrong?

Reply:
ohh sorry my bad, I was looking at the wrong line. This is right. You already have it :
https://github.com/liquibase/build-logic/pull/108/files/8e2720cda0be95876aa5eddf6decb0cf3caac164#diff-101bec72d0f1f84e7290d113531bbd8c6aa355cdc9f456df255544bc0a2da0eaR12